sqlmesh/core/console.py

```diff
@@ -300,19 +314,27 @@ def plan(self, plan: Plan, auto_apply: bool) -> None:
         if auto_apply:
             plan.apply()
 
+    def _get_invalid_tree(self, invalid_snapshot_names: t.Set[str]) -> Tree:
+        invalid = Tree(f"[bold][invalid]Invalid Snapshots:")
```

Comment:
hm re invalid, perhaps ignored? they're not really invalid entirely, the usage may be concerning